controllers/reservation.controllers.js

```diff
@@ -519,31 +517,39 @@ const submitReserve = async ({
     let day = start;
 
     while (day <= end) {
-      days.push(day.toDate());
+      const dayString = day.format('dddd');
+      if (dayString !== 'Saturday' && dayString !== 'Sunday') {
+        days.push(day.toDate());
+      }
       day = day.clone().add(1, 'd');
     }
 
     reservation = await Promise.all(days.map((_day) => verifyEachDay(_day)));
   }
 
-  await ack(errors);
-
   let message;
 
   const hasErrors = Object.keys(errors).length;
+  const roomsFullMessage = daysWithRoomFullsSelected.length
+    ? `\nOmitimos los días: ${daysWithRoomFullsSelected.map(
+      (day) => day,
+    )} porque la sala se encontraba llena`
+    : '';
 
   if (!hasErrors) {
     if (!reservation) {
       message = 'Uuups hubo un error al crear tu reserva 🙁 🥺 vuelve a internarlo más tarde';
     } else {
-      message = 'Tu reserva fue creada correctamente 🙌🏻 📩 📝';
+      message = `Tu reserva fue creada correctamente 🙌🏻 📩 📝 ${roomsFullMessage}`;
     }
   }
 
-  await client.chat.postMessage({
-    channel: body.user.id,
-    text: message,
-  });
+  if (message) {
```

Comment:
under what circumstances `message` could be falsy? and at that point, what should happen if there's no message?